packages/xo-server/src/xo-mixins/backups-ng/index.js

```diff
@@ -1018,7 +1023,14 @@ export default class BackupNg {
       .filter(_ => _.other_config['xo:backup:job'] === jobId)
       .sort(compareSnapshotTime)
 
-    xapi._assertHealthyVdiChains(vm)
+    const bypassVdiChainsCheck: boolean = getSetting(
+      settings,
+      'bypassVdiChainsCheck',
+      [vmUuid, '']
```

Comment:
Why you're passing an empty string here ?

Reply:
Because `settings` is map which associates a set of settings to an identifier, which allows us to associate different settings to different contexts.

`''` is a special id which means for all context.

`getSetting` is implemented to look for a specific for all specified contexts.

Here, we are looking for this setting for the specific VM context or for all contexts as a fallback.